OBSDOCS-1603: Add assemblies to the 'Getting started' section (#87498)
openshift/openshift-docs: observability/monitoring/getting-started/developer-and-non-administrator-steps.adoc

toc::[]

After monitoring for user-defined projects is enabled and configured, developers and other non-administrator users can then perform the following activities to set up and use monitoring for their own projects:

* xref:../../../observability/monitoring/configuring-user-workload-monitoring/configuring-metrics-uwm.adoc#setting-up-metrics-collection-for-user-defined-projects_configuring-metrics-uwm[Deploy and monitor services].
* xref:../../../observability/monitoring/managing-alerts/managing-alerts-as-a-developer.adoc#managing-alerting-rules-for-user-defined-projects-uwm_managing-alerts-as-a-developer[Create and manage alerting rules].
* xref:../../../observability/monitoring/managing-alerts/managing-alerts-as-a-developer.adoc#managing-alerts-as-a-developer[Receive and manage alerts] for your projects.

> * If granted the `alert-routing-edit` cluster role, xref:../../../observability/monitoring/configuring-user-workload-monitoring/configuring-alerts-and-notifications-uwm.adoc#configuring-alert-routing-for-user-defined-projects_configuring-alerts-and-notifications-uwm[configure alert routing].

Author: Hello @juzhao I have made one change here compared to the old list. The old one said that users need the `user-workload-monitoring-config-edit` role, however in the procedure we put the `alert-routing-edit` cluster role as a prerequisite, plus it makes sence based on the existing information about this cluster role.  So here I changed it to that, but I wanted to confirm this with you. 
Thank you!

Reviewer: yes, should use `alert-routing-edit` cluster role, verified, the doc is LGTM
see from
https://docs.openshift.com/container-platform/4.17/observability/monitoring/enabling-alert-routing-for-user-defined-projects.html#understanding-alert-routing-for-user-defined-projects_enabling-alert-routing-for-user-defined-projects
it's also `alert-routing-edit`

> * Use the {product-title} web console to xref:../../../observability/monitoring/accessing-metrics/accessing-metrics-as-a-developer.adoc#reviewing-monitoring-dashboards-developer_accessing-metrics-as-a-developer[view dashboards].

Reviewer 2: I'm not sure if you want to lead most of these steps with linked text, but if you do, you could reword this one as follows:
```suggestion
* xref:../../../observability/monitoring/accessing-metrics/accessing-metrics-as-a-developer.adoc#reviewing-monitoring-dashboards-developer_accessing-metrics-as-a-developer[View dashboards] by using the {product-title} web console.
```
